src/test/java/com/wikia/webdriver/testcases/mercurytests/PortableInfoboxTests.java

```diff
@@ -132,4 +133,40 @@ public void MercuryPortableInfoboxTest_006_DifferentClickTargets() {
         .clickGalleryButton(0)
         .isInfoboxExpanded();
   }
+
+  @Test(groups = "MercuryPortableInfoboxTest_007")
+  public void MercuryPortableInfoboxTest_007_HeroImageTall() {
+    PortableInfoboxObject infoboxTallImage = new PortableInfoboxObject(driver);
+    infoboxTallImage.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_1);
+    infoboxTallImage.isHeroImageSquare();
+  }
+
+  @Test(groups = "MercuryPortableInfoboxTest_008")
```

Comment:
By we do you mean X-wing team or who ? Sorry but I don't understand you
Do you mean that we X-wing team are running only certain singular tests and that's why they all need a separate group ?